stories/multi-combo-box/multi-combo-box.stories.js

```diff
@@ -0,0 +1,1023 @@
+export default {
+    title: 'Patterns/Multi ComboBox',
+    parameters: {
+        description: `
+The multi-combo-box component is an opinionated composition of the \`input group\`, \`popover\`, \`checkbox\`, \`list\` and \`token\` components.
+The \`popover\` is shared between the combobox and select components. Please see the \`select\` documentation for the complete list of modifiers,
+That are also  supported by the \`combobox\` component.
+
+\`Multi ComboBox\` allows users to enter multiple values which are displayed as a tokens. 
+It provides an editable input field for filtering the list, and a dropdown menu with a list of the available options. 
+If the entries are not validated by the application, users can also enter custom values. 
+`,
+        tags: ['f3', 'a11y', 'theme'],
+        components: [
+            'form-label',
+            'popover',
+            'radio',
+            'title',
+            'token',
+            'input-group',
+            'checkbox',
+            'list',
+            'button',
+            'icon',
+            'tokenizer',
+            'dialog',
+            'bar',
+            'input',
+            'layout-grid',
+            'link'
+        ]
+    }
+};
+
+export const cozyAndCompact = () => `<div class="fd-container" style="height: 300px;">
+    <div class="fd-row">
+	<div class="fd-col fd-col--6">
+        <label class="fd-form-label">
+            Multi ComboBox Cozy Mode
+        </label>
+        <div class="fd-popover">
+            <div class="fd-popover__control" role="combobox" aria-controls="F4GcX348a" aria-expanded="true" aria-haspopup="true">
+                <div class="fd-input-group fd-input-group--control" tabindex="0">
+                    <div class="fd-tokenizer">
+                        <div class="fd-tokenizer__inner">
+                            <span class="fd-token" role="button" tabindex="0">
+                                <span class="fd-token__text">
+                                    Apple
+                                </span>
+                                <button class="fd-token__close" aria-label="unselect option: apple"></button>
+                            </span>
+                            <span class="fd-token" role="button" tabindex="0">
+                                <span class="fd-token__text">
+                                    Orange
+                                </span>
+                                <button class="fd-token__close" aria-label="close button"></button>
```

Comment:
These should be updated to reflect the action of the button, rather than what the button is. Similar to the "Apple" close button being labeled "unselect option: apple", the same should be done for the rest: "unselect option: orange", "unselect option: banana".

```suggestion
                                <button class="fd-token__close" aria-label="unselect option: orange"></button>
```